ROLES.md

```diff
@@ -0,0 +1,25 @@
+# Roles
+
+## Contributor
+
+A contributor is any person who has at least 50 contributor 'Activities' in the Contributor Leaderboard for the last 12 months,
+or at least 5 pull requests merged in the last 12 months,
```

Comment:
Any pointers as to how these threshold were decided upon? 
My question is actually mostly: do we really _need_ these? Also, what's a contributor anyways? Isn't anyone who contributes, a contributor? Something that tangibly would probably result in a PR to "something"? i.e all of [these](https://github.com/Kuadrant/limitador/graphs/contributors) people for all our repos? 

Also, tho I'd generally be against such a large PR/code change, couldn't a single (or two) PRs grant a user with the request to become a contributor "with a lower threshold" based of the work that represents?

Reply:
> Any pointers as to how these threshold were decided upon?

Based on the template.

> do we really need these
Entirely up to us. 

> Also, what's a contributor anyways? Isn't anyone who contributes, a contributor?

Grammatically, sure.
It's up to us to define what's the threshold for being able to have some official power (like voting).

> couldn't a single (or two) PRs grant a user with the request to become a contributor "with a lower threshold" based of the work that represents?

Sure. Do you want to suggest some modification that captures that?